docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerHandleExample.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Button } from '@aws-amplify/ui-react';
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+
+export const StorageManagerHandleExample = () => {
+  const files = React.useRef();
+
+  return (
+    <>
+      <StorageManager
+        acceptedFileTypes={['image/*']}
+        accessLevel="public"
+        maxFileCount={3}
+        ref={files}
+      />
+      <Button onClick={() => files.current.clearFiles()}>
+        {'Clear Files'}
```

Comment:
Think this can just be
```suggestion
        Clear Files
```